src/client/binary_client.cpp

```diff
@@ -375,7 +375,7 @@ namespace
       request.MethodsToCall = methodsToCall;
       const CallResponse response = Send<CallResponse>(request);
       if (Debug) {std::cout << "binary_clinent | Call <--" << std::endl;}
-      return response.Results;
+      return response.Parameters.Results;
```

Comment:
I would actually like to not to split the structure since, it will be easier comparable with Specification (Part 4, page 62).

But how to do this? Should the "CallResult" be added in 'NoSplitStruct' list?